crypto_kem/kyber768/clean/poly.c

```diff
@@ -23,9 +23,9 @@ void PQCLEAN_KYBER768_poly_compress(unsigned char *r, const poly *a) {
             t[j] = (((PQCLEAN_KYBER768_freeze(a->coeffs[i + j]) << 3) + KYBER_Q / 2) / KYBER_Q) & 7;
         }
 
-        r[k]     =  t[0]       | (t[1] << 3) | (t[2] << 6);
-        r[k + 1] = (t[2] >> 2) | (t[3] << 1) | (t[4] << 4) | (t[5] << 7);
-        r[k + 2] = (t[5] >> 1) | (t[6] << 2) | (t[7] << 5);
+        r[k]     = (unsigned char)( t[0]       | (t[1] << 3) | (t[2] << 6));
```

Comment:
nit: the README.md says ``All integer types are of fixed size, using stdint.h types (including uint8_t instead of unsigned char)``